src/xbot/common/command/BaseRobot.java

```diff
@@ -180,14 +183,18 @@ protected void sharedPeriodic() {
         double timeSinceLastLog = Timer.getFPGATimestamp() - lastFreqCounterResetTime;
         if(lastFreqCounterResetTime <= 0) {
             lastFreqCounterResetTime = Timer.getFPGATimestamp();
+            log.info("Total Current:" + pdp.getTotalCurrent());
```

Comment:
This needs to be moved elsewhere; right now, this will only log once when the loop runs the first time.